docs/en_US/Tutorials.rst

```diff
@@ -12,4 +12,5 @@ Tutorials
     WebUI
     Training Platform<training_services>
     How to use docker<HowToUseDocker>
-    advanced
\ No newline at end of file
+    advanced
```

Comment:
I think we could change "advanced" to "Advanced"

Reply:
Well, it is related to the name of that specific rst file so if I change it here, I'll have to rename that file. Moreover, there are still some files whose name's first letter is lowercase, do you think it's necessary to change them all?

Reply:
Or we could discuss in the scrum and then make a decision.